main/guides/zoe/index.md

```diff
@@ -1,24 +1,38 @@
 # Zoe Overview
 
-## What is Zoe?
+The Zoe service and smart contract API support credibly trading assets with reduced risk.
 
-Zoe is the heart of Agoric's smart contract framework. Use Zoe to:
-
-- **Run your code on-chain**
-- **Mint new digital assets**
-- **Credibly trade assets**
-
-## Why Use Zoe?
-
-- **Zoe is Safer for Users:** A Zoe offer guarantees that you get either
+- **Zoe is Safer for Users:** Zoe guarantees that when you make an offer, you get either
   what you wanted or a full refund of the assets you put in,
   even if the contract is buggy or malicious.
 
-- **Zoe is Safer for Developers** For a given offer, if you
+- **Zoe is Safer for Developers**: For a given offer, if you
   make a mistake with the amount of assets you take or give,
   Zoe guarantees that your users will either get what they say
   they wanted or get a refund.
 
+## High Level Trading Flow
+
+Trading with a contract using Zoe typically goes through these steps:
+
+<img src="./assets/offer-safety-flow.svg" alt="offer safety flow with contracts, zoe, and parties"
+ style="border: 2px solid" />
+
+1. Parties call [E(zoe).offer(invitation, proposal, assets)](/reference/zoe-api/zoe#e-zoe-offer-invitation-proposal-paymentpkeywordrecord-offerargs). Zoe escrows the assets.
+2. Zoe relays the proposal to the contract identified in the invitation.
```

Comment:
The diagram needs an arrow for this. The diagram's **2** goes from the parties to the contract.

Reply:
Yes, that's a bit of a fudge. I'd like to leave the high level diagram to correspond exactly with the recorded talk. The diagrams in the detailed discussion have the exact arrow going from Zoe to the contract:

> Zoe gets Alice's E(zoe).offer(...) call, checks the proposal against the proposal shape, escrows the payment, and invokes the handler.
> <img src="https://docs.agoric.com/assets/trade-offer-safety-3.u_IxArtY.svg" />

Reply:
making the diagram correspond with the recorded talk is fine. can you re-word **(2)** so there's something sent directly from the parties to the contract with a parenthetical for the detail the contract verifiably gets via zoe?